root/static/scripts/release-editor/init.js

```diff
@@ -260,6 +267,73 @@ releaseEditor.init = function (options) {
     getRecordings();
   });
 
+  /**
+   * Check for similarly-named existing release groups when the release
+   * title or artist credits change.
+   */
+  let duplicateRGsRequest = null;
+  let duplicateRGsQuery = null;
+  debounceComputed(utils.withRelease((release) => {
+    const query = utils.constructLuceneFieldConjunction({
+      arid: release.artistCredit().names
+        .map((a) => a.artist?.gid)
+        .filter(Boolean),
+      releasegroup: [`"${utils.escapeLuceneValue(release.name() ?? '')}"`],
+    });
+    if (query === duplicateRGsQuery) {
+      return;
+    }
+
+    // Cancel any in-progress lookup and clear existing results.
+    duplicateRGsRequest?.abort();
+    duplicateRGsRequest = null;
+    duplicateRGsQuery = query;
+    releaseEditor.duplicateRGs.removeAll();
+
+    /*
+     * Make sure that an existing release group isn't selected
+     * and that there's a title and artist to use for searching.
+     */
+    if (
+      release.releaseGroup().gid ||
+      (release.name() ?? '') === '' ||
+      !release.artistCredit().names.some((a) => a.artist?.gid)
+    ) {
+      return;
+    }
+
+    releaseEditor.loadingDuplicateRGs(true);
+    duplicateRGsRequest = utils.search(
+      'release-group', query, maxDuplicateRGs,
+    )
+      .always(() => {
+        releaseEditor.loadingDuplicateRGs(false);
+        duplicateRGsRequest = null;
+      })
+      .done((data) => {
+        ko.utils.arrayPushAll(
+          releaseEditor.duplicateRGs,
+          data['release-groups'].map((rg) => ({
```

Comment:
I'm not sure if it's too weird to call this `duplicateRGs` when these are really minimal objects with `name`, `gid`, and localized `details` properties. On the other hand, I don't know if I'd be able to do all of the i18n stuff within `information.tt` (and I definitely don't want to :-)), and the search endpoint doesn't give all the information needed to construct a full RG object anyway.